packages/api/tests/helpers/create-test-data.js

```diff
@@ -50,6 +51,7 @@ async function createGranuleAndFiles({
   collectionCumulusId,
   esClient,
   granuleParams = { published: false },
+  writeDynamo = true,
```

Comment:
See above comment. I'd be curious what it would take to remove this completely here. it doesn't look like it's used in that many places but I'm SURE there's complexity I'm not seeing.